include/zephyr/drivers/video.h

```diff
@@ -583,49 +574,34 @@ static inline int video_flush(const struct device *dev, enum video_endpoint_id e
 }
 
 /**
- * @brief Start the video device function.
+ * @brief Start or stop the video device function.
  *
- * video_stream_start is called to enter ‘streaming’ state (capture, output...).
- * The driver may receive buffers with video_enqueue() before video_stream_start
- * is called. If driver/device needs a minimum number of buffers before being
- * able to start streaming, then driver set the min_vbuf_count to the related
+ * Start (enable == true) or stop (enable == false) streaming on the video device.
+ * The driver may receive buffers with video_enqueue() before starting streaming.
+ * If the driver/device needs a minimum number of buffers before being able to
+ * start streaming, then the driver need to set the min_vbuf_count to the related
  * endpoint capabilities.
  *
- * @retval 0 Is successful.
- * @retval -EIO General input / output error.
+ * @param dev Pointer to the device structure.
+ * @param enable If true, start streaming, otherwise stop streaming.
+ *
+ * @retval 0 on success, otherwise a negative errno code.
  */
-static inline int video_stream_start(const struct device *dev)
+static inline int video_stream_enable(const struct device *dev, bool enable)
```

Comment:
`video_stream_enable(dev, false)` is less obvious at glance than `video_stream_stop(dev)`.
There could be a `video_stream_disable(dev, false)`.

`video_flush(dev, ep, false)` will still flush even if the last argument is `false`.

Something like `video_set_stream(dev, false)` would allow to guess that the device's stream status becomes `false` (not streaming).

Maybe `video_stream_enable()` is still a better compromise?